packages/slice-machine/src/legacy/lib/models/common/widgets/Link/Form.jsx

```diff
@@ -32,6 +41,38 @@ const Form = (props) => {
           </Col>
         ))
       }
+      <Col />
+      <Col key="allowTargetBlank">
+        <Box sx={{ mt: 2 }}>
```

Comment:
two things about this:
- could we avoid the extra element? maybe placing the styling in the parent instead
- if this just adds some spacing, maybe this should be placed in the grid itself with `gap`

Reply:
@dani-mp It's there to create this gap in the grid:
<img width="989" alt="Screenshot 2024-08-27 at 08 41 17" src="https://github.com/user-attachments/assets/339da16d-07bc-4ced-a844-bf19258feeac">

I could achieve the same thing by expanding the 3rd item to 100% width, but I'd have to add another grid to keep the field at 50%. Like this:

```
Object.entries(DefaultFields).map(([key, field], index) => (
  <Col key={key} cols={index === 2 ? 1 : 2}>
    <FlexGrid>
      <Col key={key} cols={index === 2 ? 2 : 1}>
        <WidgetFormField
          ...fieldProps
        />
      </Col>
    </FlexGrid>
  </Col>
))
```

But I like that less than I like the empty `<Col />`. What do you think? Or do you have another idea of how I can do this?

Reply:
Side note: Remember that we are talking about legacy code here, we want it to be understandable but it's ok if it's not our best code as the goal is to crash it all with the redesign.

Reply:
Oh, I think I misunderstood you, sorry. 

**With a gap**
Yes, the `<Box sx={{ mt: 2 }}>` can be avoided by adding a gap to the grid. But it will mean having to update every form for every field.

**Placing the styling on the parent**
This can also be accomplished but it would mean adding a prop to the `WidgetFormField` component so that it ignores the default "130%" height for a checkbox field. That or create a whole new component for the "Allow target blank" checkbox.

None of these options seem worth it to me as I believe that we're going to rework all the field forms soon anyway. What do you think?

Reply:
by placing the margin in the parent i meant moving it to the above Col and remove the extra box

but i understand this is legacy code, most changes will be required and we don't want to invest on that atm, so you can discard this comment